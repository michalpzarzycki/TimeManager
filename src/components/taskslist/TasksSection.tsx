import React from 'react';
import TasksInfoBar from './TasksInfoBar';
import Task from './Task';
import styles from './TasksSection.module.css'

export default function TasksSection({ taskList, handleDeletePopup, isAllChecked, handleDetailsPopup}: any) {
    return(
        <section className={styles.route1TasksSection}>
        <div className={styles.route1TaskSectionTasks}>
            <TasksInfoBar handleAllChecked={() => { }} />
            <div className={styles.tasksListContainer}>
                {(!taskList.length) ?
                    <div>BRAK TASKOW</div>
                    : taskList.map((task: any) => <Task isAllChecked={isAllChecked} task={task} handleDeletePopup={handleDeletePopup} handleDetailsPopup={handleDetailsPopup} />)}
            </div>
        </div>
    </section>
    )
} 