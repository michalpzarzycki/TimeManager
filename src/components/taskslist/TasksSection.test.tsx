import React from 'react'
import {shallow} from 'enzyme'
import TasksSection from './TasksSection'

test('Expext to render TasksSection component', () => {
    expect(shallow(<TasksSection taskList={{}} handleDeletePopup={()=>{}} isAllChecked={{}} handleDetailsPopup={()=>{}}/>)).toMatchSnapshot()
})