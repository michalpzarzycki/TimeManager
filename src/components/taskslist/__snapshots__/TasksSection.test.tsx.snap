// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TasksSection component 1`] = `ShallowWrapper {}`;
